variables.tf

```diff
@@ -73,12 +73,6 @@ variable "cloud_services" {
         "cloud-object-storage",
         "codeengine",
         "container-registry",
-        "databases-for-cassandra",
-        "databases-for-elasticsearch",
-        "databases-for-enterprisedb",
-        "databases-for-mongodb",
-        "databases-for-postgresql",
-        "databases-for-redis",
```

Comment:
databases is included the list in #396 at https://cloud.ibm.com/docs/vpc?topic=vpc-vpe-supported-services